programs/rodeos/wasm_ql_http.cpp

```diff
@@ -592,27 +592,25 @@ class listener : public std::enable_shared_from_this<listener> {
    void start_listen(Acceptor& acceptor, const Endpoint& endpoint) {
       beast::error_code ec;
 
+      auto check_ec = [&](const char* what) {
+         if (!ec)
+            return;
+         elog("${w}: ${m}", ("w", what)("m", ec.message()));
+         FC_ASSERT(false, "unable to open listen socket");
```

Comment:
Can you replace "open" with `what` as check_ec can be called by `open`, `bind`, and `listen`?